modules/grafana/manifests/dashboards.pp

```diff
@@ -38,6 +38,7 @@
     "${dashboard_directory}/application_health.json": content => template('grafana/dashboards/application_health.json.erb');
     "${dashboard_directory}/edge_health.json": content => template('grafana/dashboards/edge_health.json.erb');
     "${dashboard_directory}/origin_health.json": content => template('grafana/dashboards/origin_health.json.erb');
+    "${dashboard_directory}/scheduled_publishing.json": content => template('grafana/dashboards/scheduled_publishing.json.erb');
```

Comment:
It doesn't look like this needs to be a template, so you could just add it as a file.

Reply:
Thanks, that's much easier. 👍